Makefile

```diff
@@ -14,7 +14,7 @@ GIT_VERSION = $(shell git describe --dirty --tags --always)
 GIT_COMMIT = $(shell git rev-parse HEAD)
 K8S_VERSION = v1.18.2
 GOLANGCI_LINT_VER = "1.30.0"
-OLM_VERSION = "0.15.1"
+OLM_VERSIONS = "0.16.1 0.15.1 0.14.1 0.13.0 0.12.0"
```

Comment:
I am not a big fan of we have 2 places where we define the versions. however, I think it is not a blocker at all.
just to register a few ideas, we might be done a go mod instead of using shell script such as hack/olm/tools to manage that,